client/components/GamePage.js

```diff
@@ -79,6 +80,12 @@ const useStyles = makeStyles((theme) => ({
       maxHeight: '100%',
     },
   },
+  bottomButtons: {
+    display: 'flex',
+    [theme.breakpoints.up('sm')]: {
+      maxWidth: '100%',
```

Comment:
Do we need this at all?

Reply:
I thought so, but I guess not, really.

It does change position on desktop if I remove the "display: 'flex'" though:

display: 'flex': 
<img width="1158" alt="Untitled" src="https://user-images.githubusercontent.com/570079/99341559-1d601000-2858-11eb-8921-668c32683b8e.png">

no display: 'flex': 
<img width="1161" alt="Untitled1" src="https://user-images.githubusercontent.com/570079/99341569-2650e180-2858-11eb-965d-6b92bbb1c676.png">

Reply:
CSS, how does it work?

Reply:
Oh, yeah, I was only referring to the `max-width` and the media query part.. I wish I could select code like in Gerrit :-D
Keeping `display: flex` sgtm, maybe even add `flex-wrap: wrap` so it breaks into two lines on narrow screens :-)

Reply:
> I wish I could select code like in Gerrit :-D

yeah, that would be nice. I wouldn't be surprised if there's a Chrome extension or something that enables that.